packages/amplify_auth_cognito/example/ios/unit_tests/amplify_auth_cognito_tests.swift

```diff
@@ -1109,6 +1107,37 @@ class amplify_auth_cognito_tests: XCTestCase {
         })
     }
     
+    func test_fetchUserAttributesSuccess() {
+        class FetchAttributesMock: AuthCognitoBridge {
+            override func onFetchUserAttributes(flutterResult: @escaping FlutterResult) {
+                let emptyList = [AuthUserAttribute]()
+                flutterResult(emptyList)
+            }
+        }
+        
+        plugin = SwiftAuthCognito.init(cognito: FetchAttributesMock())
+        
+        _data = [:]
+        _args = ["data": _data]
+        let call = FlutterMethodCall(methodName: "fetchUserAttributes", arguments: _args)
+        plugin.handle(call, result: {(result)->Void in
+            if let res = result as? [AuthUserAttribute] {
+                XCTAssertEqual( 0, res.count )
+            } else {
+                XCTFail()
+            }
```

Comment:
Should we not actually create or mock a userAttribute in this test?

Reply:
Fixed.